power_grid_model_c/power_grid_model/include/power_grid_model/sparse_mapping.hpp

```diff
@@ -7,6 +7,7 @@
 #define POWER_GRID_MODEL_SPARSE_MAPPING_HPP
 
 #include "power_grid_model.hpp"
+using namespace std;
```

Comment:
please never ever ever do `using namespace std;` in a header file 😬 it may mess up your entire environment

the only exception may be if it is used in another namespace (and even then, be veeeerrrryyyyy careful when you do)

if you really need some functionality from `std` and want to expose it globally as if it were your own function, you can do `using std::func;` but please only do this if you know what you're doing

Reply:
Okay, this will be changed together with the sorting algorithm, since I only wanted to use it to test the sorting as well as the unit test. Thank you for letting me know!